Formula/go.rb

```diff
@@ -20,6 +20,15 @@ class Go < Formula
     sha256 "4ae792ffe880c4658a596a82fa8276d39d78d2e7b64e7ac9598649f0eefed90c" => :el_capitan
   end
 
+  devel do
+    url "https://redirector.gvt1.com/edgedl/go/go1.10beta1.src.tar.gz"
```

Comment:
What's with the URL change?

Reply:
I just yanked the URL from what's on https://golang.org/dl now. `whois gvt1.com` suggests that Google owns it. I'll see if I can dig up some docs on the change.

Reply:
Ok, they [appear to be official](https://github.com/golang/go/issues/22648), but it looks like they don't really want them exposed. This works:

```➜  ~ curl -I https://dl.google.com/go/go1.10beta1.src.tar.gz
HTTP/2 200 
accept-ranges: bytes
cache-control: public,max-age=86400
content-length: 18312654
content-type: application/octet-stream
etag: "197f00"
last-modified: Thu, 07 Dec 2017 22:43:43 GMT
server: downloads
x-content-type-options: nosniff
x-frame-options: SAMEORIGIN
x-xss-protection: 1; mode=block
date: Fri, 08 Dec 2017 00:29:50 GMT
alt-svc: hq=":443"; ma=2592000; quic=51303431; quic=51303339; quic=51303338; quic=51303337; quic=51303335,quic=":443"; ma=2592000; v="41,39,38,37,35"```

Shall I swap that one in?

Reply:
cc @bradfitz

Reply:
Yeah, it's a Google URL; it's the one that gets used when Chrome does autoupdate checking & finds an update available as well. Just unusual to see it being used as the public-facing URL rather than as a redirect from elsewhere, but the change seems legit.

Reply:
`https://golang.org/dl/go1.9.2.src.tar.gz` works with stable as well. Homebrew follows redirects (via `curl`) obviously, and informs the user about the redirect, so that would seem fine to me at least if Go upstream are ditching the old URL.

```
~> brew fetch go -s -v
==> Downloading https://golang.org/dl/go1.9.2.src.tar.gz
==> Downloading from https://redirector.gvt1.com/edgedl/go/go1.9.2.src.tar.gz
```

Reply:
Use https://dl.google.com/go/go1.9.2.src.tar.gz which has no redirects and is a pretty URL.

Reply:
@ilovezfs Any issues changing both the stable/devel/resource URLs here or do you want it phased in as updates to become available?

Reply:
@DomT4 just pushed the devel change. Happy to update the stable URL if @ilovezfs says to.

Reply:
Let's phase them in as updates come available. I'm sure @DomT4 won't let anyone forget :)

Reply:
Your faith in me is... concerning 😉.